econplayground/assignment/models.py

```diff
@@ -177,18 +177,15 @@ class Step(MP_Node):
         Question, on_delete=models.SET_NULL,
         blank=True, null=True)
 
-    next_step = models.ForeignKey(
```

Comment:
next_step represents the next step override configured by the instructor:
![Screenshot_2023-11-20_14-09-46](https://github.com/ccnmtl/econplayground/assets/59292/6a68b52a-739d-4d1b-b1bc-a1b3dd2a2271)

We don't want to remove this.

Reply:
Ohh it's an override. Understood.

Reply:
Yeah, sorry - that should be documented better here! I'll add a comment.